clients/browser/index.html

```diff
@@ -404,7 +404,7 @@
 
         _balanceChanged(balance) {
             if (!balance) balance = Nimiq.Balance.INITIAL;
-            this._wltBalance.innerText = Nimiq.Policy.satoshisToCoins(balance.value).toFixed(Math.log10(Nimiq.Policy.SATOSHIS_PER_COIN));
+            this._wltBalance.innerHTML = '<span class="icon-nimiq"></span> ' + Nimiq.Policy.satoshisToCoins(balance.value).toFixed(Math.log10(Nimiq.Policy.SATOSHIS_PER_COIN));
```

Comment:
I would prefer keeping `innerText` here and not putting the icon there on every update.
Instead, you could hardcode the icon into the HTML right before the balance, e.g., as follows:
```
<span class="icon-nimiq"></span> <span id="wltBalance">0.00000000</span>
```